evalml/model_understanding/graphs.py

```diff
@@ -569,6 +572,90 @@ def graph_prediction_vs_actual(y_true, y_pred, outlier_threshold=None):
     return _go.Figure(layout=layout, data=data)
 
 
+def get_tree_data(clf):
```

Comment:
@ParthivNaresh a naming pattern we've followed with the other model understanding tools is for the data method to be called "{}" and then for the graphing method which uses that data to be called "graph_{}". Should we follow the same pattern here? I.e., rename `get_tree_data` to `decision_tree_data`. I think its fine to keep `visualize_decision_tree` as-is.

Reply:
@dsherry That sounds good. Since I'm implementing Freddy's recommendation for reading from a Pipeline as well, it could follow `decision_tree_data_from_estimator` and `decision_tree_data_from_pipeline`.